src/lints/trait_removed_associated_type.ron

```diff
@@ -0,0 +1,54 @@
+SemverQuery(
+    id: "trait_removed_associated_type",
+    human_readable_name: "associated type of trait removed",
+    description: "A trait's associated type was removed.",
```

Comment:
```suggestion
    description: "A trait's associated type was removed or renamed.",
```
to make it in line with our other explanations for removed items

Reply:
Yup, makes perfect sense